quickwit-doc-mapper/src/tokenizers.rs

```diff
@@ -17,14 +17,122 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with this program. If not, see <http://www.gnu.org/licenses/>.
 
+use std::str::CharIndices;
+
 use once_cell::sync::Lazy;
-use tantivy::tokenizer::{RawTokenizer, RemoveLongFilter, TextAnalyzer, TokenizerManager};
+use regex::Regex;
+use tantivy::tokenizer::{
+    BoxTokenStream, RawTokenizer, RemoveLongFilter, TextAnalyzer, Token, TokenStream, Tokenizer,
+    TokenizerManager,
+};
+
+static REGEX_ARRAY: [Lazy<Regex>; 3] = [
```

Comment:
You can wrap the whole array
```suggestion
static REGEX_ARRAY: Lazy<[Regex; 3]> = [
```